include/cuco/detail/open_addressing/functors.cuh

```diff
@@ -18,6 +18,8 @@
 #include <cuco/detail/bitwise_compare.cuh>
 #include <cuco/detail/pair/traits.hpp>
 
+#include <thrust/tuple.h>
```

Comment:
I think `cuda::std::tuple` should be preferred here? Please try it. It's also okay to defer this to another PR.

Reply:
Right, the issue is from the outdated CCCL version, where `zip_iterator` is incompatible with `cuda::std::tuple`: https://godbolt.org/z/jnKafGPf4. Currently, cuco is still pulling in `rapids-cmake` branch-25.02, which brings in CCCL 2.7. Once we update to a newer version of `rapids-cmake`, CCCL will be upgraded accordingly, and we’ll be able to fully replace `thrust::tuple` with `cuda::std::tuple` in cuco.